.travis.yml

```diff
@@ -9,6 +9,7 @@ before_install:
   - sudo curl https://github.com/mikefarah/yq/releases/download/3.2.1/yq_linux_amd64 -L -o /usr/local/bin/yq-3.2.1
   - sudo chmod +x /usr/local/bin/yq-3.2.1
   - sudo ln -s /usr/local/bin/yq-3.2.1 /usr/local/bin/yq
+  - sudo apt install shellcheck
```

Comment:
Hm hm can we just use one `apt install ...` step ? i.e. add `shellcheck` to line 7?

Reply:
TBH I thought about it but I like to have separate commands in separate lines (so that it's obvious to spot the diff) and I'm not sure I'm allowed to use multiline calls in here.